packages/@react-spectrum/s2/stories/ActionButton.stories.tsx

```diff
@@ -198,3 +200,17 @@ export const Avatars: Story = {
     );
   }
 };
+
+export const NotificationBadges: Story = {
+  render: (args) => {
+    return (
+      <div style={{display: 'flex', gap: 8, padding: 8, justifyContent: 'center'}}>
+        <ActionButton aria-label="Press me" {...args}><NewIcon /><NotificationBadge /></ActionButton>
```

Comment:
ah, i see. we still want the notification to be announced I'm pretty sure. so we should add some logic in ActionButton, if an aria-label is provided, then we need to add an aria-labelledby= self + notificationbadge

`useLabel` or `useLabels` should handle the logic for you, I forget which one off the top of my head